test/correctness/predicated_store_load.cpp

```diff
@@ -376,7 +367,14 @@ int vectorized_predicated_load_const_index_test() {
     return 0;
 }
 
-int vectorized_predicated_load_lut_test() {
+int vectorized_predicated_load_lut_test(const Target &t) {
+    if (t.arch != Target::X86) {
```

Comment:
If the failure is specific to HVX, shouldn't we test for that instead? (ie, allow it if testing on arm, etc)

Reply:
The failure is because we're using 32 bit LUT. If I make the LUT into 16-bit, the simplifier will remove the predicate and we ended up not having the predicated vector any longer. Also, the predicated vector right now only happens on Hexagon and X86, so there is no point on testing it on arm etc.